src/main/java/net/gudenau/minecraft/moretags/mixins/block/farmland/PlantBlockMixin.java

```diff
@@ -1,29 +1,23 @@
 package net.gudenau.minecraft.moretags.mixins.block.farmland;
 
 import net.gudenau.minecraft.moretags.MoreBlockTags;
-import net.minecraft.block.Block;
 import net.minecraft.block.BlockState;
-import net.minecraft.block.Blocks;
 import net.minecraft.block.PlantBlock;
-import net.minecraft.tag.BlockTags;
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.world.BlockView;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
 
-@Mixin(PlantBlock.class)
+@Mixin(value = PlantBlock.class, priority = 1001)
 public abstract class PlantBlockMixin {
     @Inject(
         method = "canPlantOnTop",
-        at = @At(
-            value = "INVOKE",
-            target = "Lnet/minecraft/block/BlockState;isOf(Lnet/minecraft/block/Block;)Z"
-        ),
+        at = @At("RETURN"),
         cancellable = true
     )
     private void canPlantOnTop(BlockState floor, BlockView world, BlockPos pos, CallbackInfoReturnable<Boolean> cir) {
-        cir.setReturnValue(floor.isIn(MoreBlockTags.PLANTABLE_BLOCKS));
+        cir.setReturnValue(cir.getReturnValue() || floor.isIn(MoreBlockTags.FARMLAND));
```

Comment:
Is the change from plantable_blocks to farmland intended here ?

Reply:
Unintended. Good catch